bin/citrea/tests/e2e/mod.rs

```diff
@@ -2065,10 +2065,10 @@ async fn sequencer_crash_and_replace_full_node() -> Result<(), anyhow::Error> {
 
     let commitments = db_test_client.get_all_commitments().await.unwrap();
     assert_eq!(commitments.len(), 2);
-    assert_eq!(commitments[0].l1_start_height, 1);
-    assert_eq!(commitments[0].l1_end_height, 1);
-    assert_eq!(commitments[1].l1_start_height, 2);
-    assert_eq!(commitments[1].l1_end_height, 3);
+    // assert_eq!(commitments[0].l1_start_height, 1);
+    // assert_eq!(commitments[0].l1_end_height, 1);
+    // assert_eq!(commitments[1].l1_start_height, 2);
+    // assert_eq!(commitments[1].l1_end_height, 3);
```

Comment:
why?

Reply:
I thought we commmented out L2 comparison :D

let's compare l2 heights here instead.